components/authz-service/engine/opa/opa.go

```diff
@@ -127,26 +127,26 @@ func (s *State) initModules() error {
 
 // initPartialResult allows caching things that don't change among multiple
 // query evaluations. We don't bother for the pairs query, but for
-// IsAuthorized(), we want to do as little work per call as possible.
-func (s *State) initPartialResult(ctx context.Context) error {
-	// Reset compiler to avoid state issues
-	compiler, err := s.newCompiler()
-	if err != nil {
-		return err
-	}
-
-	r := rego.New(
-		rego.ParsedQuery(s.queries[authzQuery]),
-		rego.Compiler(compiler),
-		rego.Store(s.store),
-	)
-	pr, err := r.PartialResult(ctx)
-	if err != nil {
-		return errors.Wrap(err, "partial eval")
-	}
-	s.partialAuth = pr
-	return nil
-}
+// SetPolicies(), we want to do as little work per call as possible.
```

Comment:
Why is this commented out instead of deleted?

Reply:
Should be deleted. Forgot to. Thanks!